packages/@aws-cdk/aws-cloudwatch/lib/dashboard.ts

```diff
@@ -75,6 +75,11 @@ export interface DashboardProps {
  * A CloudWatch dashboard
  */
 export class Dashboard extends Resource {
+  /**
+   * The name of this queue
+   * @attribute
```

Comment:
* The type is `string` as it should be populated by the physical name property (see below)
* "queue" probably a copy paste error